subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/modifiers/ModularizationOption.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.buildinit.plugins.internal.modifiers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum ModularizationOption implements WithIdentifier {
+    SINGLE_PROJECT("Single project"),
```

Comment:
Maybe instead of "Single project" we could say "All functionality in one project" 

And instead of "Add library projects" we could say "Functionality split into multiple projects"

Reply:
What about this:

```
Split functionality across multiple subprojects?:
   1: No (only one application project)
   2: Yes (application and library projects)
```

Reply:
+1

Reply:
Follow up: https://github.com/gradle/gradle/pull/14475/files#diff-5f85dd977adf6a79dcb929edacdfd13b